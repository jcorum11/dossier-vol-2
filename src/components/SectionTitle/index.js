import styled from 'styled-components'
import { colors } from '../../variables'

const SectionTitle = ({ name = '' }) => {
  return (
    <Title data-testid={name}>{name.toUpperCase()}</Title>
  )
}

const Title = styled.h1`
margin: 0;
text-align: center;
font-size: 10rem;
color: ${colors.pink};
background-color: ${colors.blue};
letter-spacing: 1rem;
scroll-snap-align: start;
@media screen and (max-width: 786px) {
  font-size: 2rem;
}
`

export default SectionTitle
