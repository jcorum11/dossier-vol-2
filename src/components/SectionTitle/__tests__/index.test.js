import { render, cleanup } from '@testing-library/react'
import SectionTitle from '..'

afterEach(cleanup)

describe('SectionTitle', () => {
  it('renders', async () => {
    render(<SectionTitle />)
  })
  it('matches snapshot', () => {
    const {asFragment} = render(<SectionTitle />)
    expect(asFragment()).toMatchSnapshot()
  })
})