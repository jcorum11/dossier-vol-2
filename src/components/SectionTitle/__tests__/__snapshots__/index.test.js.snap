// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <h1
    class="sc-bdvvtL eCmgPR"
    data-testid=""
  />
</DocumentFragment>
`;
